cmd/minishift/cmd/start.go

```diff
@@ -696,6 +701,7 @@ func initStartFlags() *flag.FlagSet {
 	startFlagSet.String(configCmd.SSHKeyToConnectRemote.Name, "", "SSH private key location on the host to connect remote machine")
 	startFlagSet.String(configCmd.ISOUrl.Name, minishiftConstants.CentOsIsoAlias, "Location of the minishift ISO. Can be a URL, file URI or one of the following short names: [centos b2d].")
 	startFlagSet.String(configCmd.TimeZone.Name, constants.DefaultTimeZone, "TimeZone for Minishift VM")
+	startFlagSet.Bool(configCmd.RegistryLogin.Name, false, "Login to registry.redhat.io")
```

Comment:
Here it should be mentioned from where the login is taken. I believe it honors env variable MINISHIFT_USERNAME and MINISHIFT_PASSWORD.

Reply:
and here a fixed registry is mentioned, so I would expect that to be somehow in the name.